packages/compose/src/hooks/use-callback-ref/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import memize from 'memize';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+function useCallbackRef( callback, deps ) {
+	return useCallback(
```

Comment:
With memize, this is no longer necessary, right?

Reply:
it is necessary in case we want to trigger the callback again if a dependency change.